falco/README.md

```diff
@@ -19,163 +17,89 @@ helm repo update
 
 ## Installing the Chart
 
-To install the chart with the release name `falco` run:
+To install the chart with the release name `falco` in namespace `falco` run:
 
 ```bash
-helm install falco falcosecurity/falco
+kubectl create ns falco
+helm install falco falcosecurity/falco --namespace falco
 ```
 
-After a few seconds, Falco should be running.
+After a few minutes Falco instances should be running on all your nodes. The status of Falco pods can be inspected through *kubectl*:
+```bash
+kubectl get pods -n falco -o wide
+```
+If every thing went smoothly, you should observe an output similar to the following, indicating that all Falco instances are up and running in you cluster:
 
+```bash
+NAME          READY   STATUS    RESTARTS   AGE     IP          NODE            NOMINATED NODE   READINESS GATES
+falco-57w7q   1/1     Running   0          3m12s   10.244.0.1   control-plane   <none>           <none>
+falco-h4596   1/1     Running   0          3m12s   10.244.1.2   worker-node-1   <none>           <none>
+falco-kb55h   1/1     Running   0          3m12s   10.244.2.3   worker-node-2   <none>           <none>
+```
+The cluster in our example has three nodes, one *control-plane* node and two *worker* nodes. The default configuration in `values.yaml` of our helm chart deploys Falco using a `daemonset`. That's the reason why we have one Falco pod in each node. 
 > **Tip**: List all releases using `helm list`, a release is a name used to track a specific deployment
 
-### About the driver
+### Falco, Event Sources and Kubernetes
+Starting from Falco 0.31.0 the [new plugin system](https://falco.org/docs/plugins/) is stable and production ready. The **plugin system** can be seen as the next step in the evolution of Falco. Historically, Falco monitored system events from the **kernel** trying to detect malicious behavior on Linux systems. It also had the capability to process k8s Audit Logs to detect suspicious activity in kubernetes clusters. Since Falco 0.32.0 all the related code to the k8s Audit Logs in Falco was removed and ported in a [plugin](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit). At the time being Falco supports different event sources coming from **plugins** or the **drivers** (system events). 
```

Comment:
```suggestion
Starting from Falco 0.31.0 the [new plugin system](https://falco.org/docs/plugins/) is stable and production ready. The **plugin system** can be seen as the next step in the evolution of Falco. Historically, Falco monitored system events from the **kernel** trying to detect malicious behaviors on Linux systems. It also had the capability to process k8s Audit Logs to detect suspicious activities in kubernetes clusters. Since Falco 0.32.0 all the related code to the k8s Audit Logs in Falco was removed and ported in a [plugin](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit). At the time being Falco supports different event sources coming from **plugins** or the **drivers** (system events). 
```